templates/etc/httpd/conf.d/pulp_docker.conf.erb

```diff
@@ -4,8 +4,21 @@
 
 # -- HTTPS Repositories ---------
 
-Alias /pulp/docker /var/www/pub/docker/web
+# This prevents mod_mime_magic from adding content-type and content-encoding headers, which will confuse the Docker
+# client.
+MimeMagicFile NEVER_EVER_USE
 
-<Location /var/www/pub/docker/web>
+# Docker v2
+Alias /pulp/docker/v2 /var/www/pub/docker/v2/web
+<Directory /var/www/pub/docker/v2/web>
+    Header set Docker-Distribution-API-Version "registry/2.0"
+    SSLRequireSSL
+    Options FollowSymlinks Indexes
+</Directory>
+
+# Docker v1
```

Comment:
My understanding is that in Pulp's realm, repos are not associated with any particular content type. Thus, you can mix v1 and v2 content in the same repo even though they work with different units (images vs blobs).

In practice, it would be extremely complicated to support this type of arrangement, but IMO the puppet module may as well expose the directory for users of this puppet module besides Katello.